ext/travisci/pgbin-from-spec

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from sys import exit, stderr
+import os, platform, sys
+
+def usage(stream):
+    print('Usage: pgbin-from-spec core/openjdk8/pg9.6', file=stream)
+
+def misuse():
+    usage(stderr)
+    exit(2)
+        
+len(sys.argv) == 2 or misuse()
+specs = sys.argv[1]
+specs = specs.split('/')
+specs = [x for x in specs if x.startswith('pg-')]
+len(specs) == 1 or misuse()
```

Comment:
We could save ourselves some duplication if we got the pg version using prefixed-ref-from-spec and then just did the os specific stuff in this file.

Reply:
We could, but this seemed simple enough to be worth avoiding the external dep (finding the other tool wrt PATH or script-home or...), and not worth forking a second process.